### DISCUSSION TITLE
更新后，alist无法显示目录

### DISCUSSION THREAD

**Discussion by NekoMirra** [AlistGo/alist#1409] [Q&A]
更新后除了自定义样式什么都没了，白屏，管理页面打开也是白屏，试图进入管理页面也进不去。。
![image](https://user-images.githubusercontent.com/48759390/181221539-00a3dcfa-f7ae-4531-9fe5-10dd96644d8d.png)

**Comment by forest-devil** [comment]
同白屏，控制台显示以下两个js资源报403无权限错误，不知道是不是这个引起的：
https://npm.elemecdn.com/alist-web@2.6.2/dist/assets/vendor.dd75dc04.js
https://npm.elemecdn.com/alist-web@2.6.2/dist/assets/index.98f0b294.js

参考响应：
```
<Error>
<Code>AccessDenied</Code>
<Message>You have no right to access this object because of bucket acl.</Message>
<RequestId>62E1501D2979B13531CD22E5</RequestId>
<HostId>fe-static-zbprod-zb1-oss-3.oss-cn-zhangjiakou.aliyuncs.com</HostId>
</Error>
```
其中RequstId每次都会变化

**Comment by nitsort** [comment]
https://github.com/alist-org/alist/discussions/1357
